extensions/amp-analytics/0.1/vendors.js

```diff
@@ -458,6 +459,16 @@ export const ANALYTICS_CONFIG = {
     },
   },
 
+  'adobeanalytics_iframePing': {
```

Comment:
@cramforce,

No problem making this change but am a little unsure of your ask. Are you asking for "adobeanalytics_iframeView", or "adobeanalytics_view"?